modules/ROOT/pages/fd-project-history.adoc

```diff
@@ -0,0 +1,27 @@
+= Discover What’s Changed in a Project
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords:
+
+:imagesdir: ../assets/images
+
+You can see a history of changes you make to a project in Flow Designer.
+The saved states are listed by time/date and activity, and their name depends on the activity you were doing at the time the save occurred.
+
+Being able to view a saved state of a project is useful if you want to:
+
+* Detect what is or is not working between two different versions. For example, you started with a working, tested flow but you see errors and you want to revert back to the working state.
```

Comment:
I would add things like:
- Check the changes made by another user on the project.
- Retrieve older values of some configured cards.

There are a couple of things we could think of, @msiqu, @ryanhol do you think we could add other things here?